src/newrelic/processor_test.go

```diff
@@ -1409,13 +1421,18 @@ func runMockedCollectorHarvestLimitTest(t *testing.T, eventType string, agentLim
 	//       So the actual value used to compare to the collector
 	//       limit will be 1/12th (5s/60s) smaller
 	var appInfo AppInfo = AppInfo{}
+
+	// defaults that collector will send if no harvest limit requested in connect request
 	logHarvestLimit := 833
+	customHarvestLimit := 2500
 
 	switch eventType {
 	case "log_event_data":
 		logHarvestLimit = int(collectorLimit)
 		appInfo = NewAppInfoWithLogEventLimit(int(agentLimit))
-
+	case "custom_event_data":
```

Comment:
Can we update the comment on line 1411 to reflect this addition?

Reply:
Updated in commit  [c1c0d8](https://github.com/newrelic/newrelic-php-agent/pull/584/commits/c1c0d8a61f54707563418906303dd952ecbfd592).